modules/installation-installer-provisioned-vsphere-config-yaml.adoc

```diff
@@ -114,18 +116,20 @@ You can specify the path of any datastore that exists in a datastore cluster. By
 If you must specify VMs across multiple datastores, use a `datastore` object to specify a failure domain in your cluster's `install-config.yaml` configuration file. For more information, see "VMware vSphere region and zone enablement".
 ====
 <7> Optional: Provides an existing resource pool for machine creation. If you do not specify a value, the installation program uses the root resource pool of the vSphere cluster.
-<8> The vSphere disk provisioning method.
+<8> Optional: Each VM created by {product-title} is assigned a unique tag that is specific to the cluster. The assigned tag enables the installation program to identify and remove the associated VMs when a cluster is decommissioned. You can list of up to ten tag IDs.
```

Comment:
```suggestion
<8> Optional: Each VM created by {product-title} is assigned a unique tag that is specific to the cluster. The assigned tag enables the installation program to identify and remove the associated VMs when a cluster is decommissioned. You can list of up to ten additional tag IDs to be attached to the VMs provisioned by the installer.
```

Reply:
@rvanderp3 can you approve if the changes look fine now?